src/zeek.bif

```diff
@@ -5103,14 +5103,18 @@ function anonymize_addr%(a: addr, cl: IPAddrAnonymizationClass%): addr
 ##                       rendered name. The default pattern strips a leading
 ##                       underscore.
 ##
+## interval_as_double: If true, interval values will be logged as doubles
+##                     instead of the broken-out version with units.
```

Comment:
Would be good to mention type string. Maybe this?
```suggestion
##                     instead of the broken-out version with units as strings.
```